modules/reference/pages/properties/object-storage-properties.adoc

```diff
@@ -85,7 +85,7 @@ AWS or GCS bucket that should be used to store data.
 
 Max size of object storage cache.
 
-If both this property and <<cloud_storage_cache_size_percent>> are set, Redpanda uses the minimum of the two.
+If both this property and <<cloud_storage_cache_size_percent>> are set, Redpanda uses the minimum of the two. If one of them is set to 0, Redpanda uses the non-zero one. If they are both set to 0, the cache size is 0.
```

Comment:
any resolution on this @jcipar @daisukebe ?

Reply:
Given there's no guardrails on setting to zero from preventing, we should document like below?

```
If both this property and <<cloud_storage_cache_size_percent>> are set, Redpanda uses the minimum of the two. If one of them is set to 0, Redpanda uses the non-zero one. Redpanda strongly recommends you do not set both of them to zero.
```

@jcipar @WillemKauf ^

Reply:
@jcipar and @WillemKauf Looks like this PR is stuck and awaiting your guidance here. Please respond asap. Thx.

Reply:
Hi @Feediver1, [I just put out a PR](https://github.com/redpanda-data/redpanda/pull/23337) that will update the behaviour here and (hopefully) make the update to the docs more straight-forward. We will now prevent the following invalid configurations:

1. `cloud_storage_cache_size` and `cloud_storage_cache_size_percent` cannot both be 0
2. `cloud_storage_cache_size` cannot be `0` while `cloud_storage_cache_size_percent` is `nil` (`std::nullopt`)

I would say something like:

```
If both this property and <<cloud_storage_cache_size_percent>> are set, Redpanda uses the minimum of the two. If one of them is set to 0, Redpanda uses the non-zero one. These properties cannot both be `0` simultaneously. 
```

Hope this helps.

Reply:
@WillemKauf @jcipar - The description has now been updated to reflect the change in the code. Please could you take a look and approve.

Also, please provide guidance on what happens when both properties are set to `0` so that we can provide troubleshooting advice where necessary.